packages/lesswrong/components/common/ContentItemBody.tsx

```diff
@@ -142,12 +157,13 @@ class ContentItemBody extends Component<ContentItemBodyProps,ContentItemBodyStat
     }
   }
   
+  
```

Comment:
This is a nit, but I counted 12 different places in this PR where we add a random new line or trailing space. Obviously it's not the end of the world but it'd be nice to not do that.

I think there's also two lines that I saw with commented out code. Ideally that should also be removed, or if we want to keep it around for the future I think there should probably be a comment with it saying why otherwise it'll just be deleted by the next person to edit that file.